crates/aptos/src/common/init.rs

```diff
@@ -440,7 +440,7 @@ impl InitTool {
 /// Any command using this, will be simpler to setup as profiles
 #[derive(Copy, Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
 pub enum Network {
-    Mainnet,
+    // Mainnet,
```

Comment:
A more elegant way to remove items is with conditional compilation:
```suggestion
    #[cfg(feature = "aptos")]
    Mainnet,
```

But commenting out is simpler (you'd have to go and conditionalize all the output string literals the same way), so it will do.